sinks/cache/cache_impl.go

```diff
@@ -21,16 +21,19 @@ import (
 
 	source_api "github.com/GoogleCloudPlatform/heapster/sources/api"
 	"github.com/GoogleCloudPlatform/heapster/store"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 )
 
 const rootContainer = "/"
 
 type containerElement struct {
+	lastUpdated time.Time
 	Metadata
 	metrics store.TimeStore
 }
 
 type podElement struct {
+	lastUpdated time.Time
```

Comment:
I assume you probably already checked, but can you please double check that this is set  appropriately everywhere that the pod/container elements are updated?